x/swaprouter/keeper.go

```diff
@@ -12,21 +13,21 @@ type Keeper struct {
 	storeKey sdk.StoreKey
 	// TODO: remove nolint
 	// nolint: unused
-	gammKeeper types.SwapI
+	gammKeeper types.SimulationExtension
```

Comment:
same question, how about SwapExtension or GammSwapExtension?

Reply:
This is needed for simulator only. I created an issue to refactor this later